extra_requirements.txt

```diff
@@ -3,4 +3,5 @@ tensorboard
 tensorboardX
 colorama
 torchvision
-pandas
\ No newline at end of file
+pandas
+notif
```

Comment:
I don't think we should add notif as an extra requirements. I think that the callback should specify the expected interface of the notificator and you could give your notif library as an example of library that implement the said interface.

Reply:
I added an interface.